### ISSUE TITLE
Precompilation warning: could not import `TimeSteppers.store_tendencies!` into `Sediments`

### ISSUE THREAD

**Issue by ali-ramadhan** [OceanBioME/OceanBioME.jl#277]
Pre-compiling OceanBioME.jl produces this warning:
```
  1 dependency had output during precompilation:
┌ OceanBioME
│  WARNING: could not import TimeSteppers.store_tendencies! into Sediments
└  
```

I think `store_tendencies!` is now called `cache_previous_tendencies!`.

---

This is happening on the current `main` branch (commit https://github.com/OceanBioME/OceanBioME.jl/tree/14ad3b6421b6946ee462f6be38ac2b5d2c2ca429) with:

```

julia> versioninfo()
Julia Version 1.10.9
Commit 5595d20a287 (2025-03-10 12:51 UTC)
Build Info:
  Official https://julialang.org/ release
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: 48 × AMD Ryzen Threadripper 7960X 24-Cores
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-15.0.7 (ORCJIT, znver3)
Threads: 1 default, 0 interactive, 1 GC (on 48 virtual cores)
Environment:
  LD_PRELOAD = /usr/NX/lib/libnxegl.so
```

```
(OceanBioME) pkg> st
Project OceanBioME v0.14.2
Status `~/atdepth/OceanBioME.jl/Project.toml`
  [79e6a3ab] Adapt v4.3.0
  [a9b6321e] Atomix v1.1.2
  [052768ef] CUDA v5.8.3
  [9a22fb26] GibbsSeaWater v0.1.4
  [033835bb] JLD2 v0.5.15
  [63c18a36] KernelAbstractions v0.9.38
  [9e8cae18] Oceananigans v0.97.8
  [6fe1bfb0] OffsetArrays v1.17.0
  [f2b01f46] Roots v2.2.10
  [d496a93d] SeawaterPolynomials v0.3.10

```
